pennylane_lightning/lightning_qubit/lightning_qubit.py

```diff
@@ -374,25 +374,26 @@ def _apply_lightning_controlled(self, sim, operation):
                 target_wires = list(set(self.wires.indices(operation.wires)) - set(control_wires))
             else:
                 target_wires = self.wires.indices(operation.target_wires)
-            paulix = getattr(sim, "PauliX", None)
-            for wire, value in zip(control_wires, control_values):
-                if not value:
-                    paulix([wire], False, [])
             if method is not None:  # apply n-controlled specialized gate
                 inv = False
                 param = operation.parameters
-                method(control_wires, target_wires, inv, param)
+                method(control_wires, control_values, target_wires, inv, param)
```

Comment:
This line is not been covered by tests. Could you please look into that?

Reply:
Yes. [Here](https://app.codecov.io/gh/PennyLaneAI/pennylane-lightning/pull/576/blob/pennylane_lightning/lightning_qubit/lightning_qubit.py#L380) it reports proper coverage. The line should be hit by this [test](https://github.com/PennyLaneAI/pennylane-lightning/blob/add_ctrl_values/tests/test_gates.py#L368).

Reply:
Thanks!